samples/snippets/pom.xml

```diff
@@ -100,7 +100,9 @@
       <version>1.1.3</version>
       <scope>test</scope>
     </dependency>
+  <!-- [START bigquery_java_dependencies] -->
```

Comment:
This isn't the snippet that is used in the README, you want the `bigquery_install_with_bom` one

Reply:
done. thanks!